apps/test/unit/lib/levelbuilder/code-docs-editor/MethodTokenTest.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {expect} from '../../../../util/reconfiguredChai';
+import MethodToken from '@cdo/apps/lib/levelbuilder/code-docs-editor/MethodToken';
+import sinon from 'sinon';
+
+describe('MethodToken', () => {
+  let defaultProps, updateSpy;
+
+  beforeEach(() => {
+    updateSpy = sinon.spy();
+    defaultProps = {
```

Comment:
I'm curious why you assign the `defaultProps` in the `beforeEach` block

Reply:
Ah I usually put the default props at the top of the file, is there a benefit to putting them in `beforeEach`?

Reply:
No - feel free to leave it, I was just genuinely curious!